_data/gaming.yml

```diff
@@ -181,20 +181,20 @@ websites:
       exceptions:
           text: "Software implementation requires you to have purchased either Final Fantasy XI or Final Fantasy XIV and registered the game."
 
+    - name: "Star Wars: The Old Republic"		
+      url: http://www.swtor.com/		
+      img: swtor.png		
+      tfa: Yes		
+      software: Yes		
+      hardware: Yes		
+      doc: http://www.swtor.com/info/security-key		
+ 
```

Comment:
Can and will clean.

I'm also looking into using rubocop or some other sort of lint tool to
enforce the editorconfig rules (so that I don't make mistakes like this)

On Wed, Aug 24, 2016 at 3:46 PM mxxcon <notifications@github.com> wrote:

> In _data/gaming.yml
> <https://github.com/2factorauth/twofactorauth/pull/1925#discussion_r76153542>
> :
>
> > @@ -181,20 +181,20 @@ websites:
> >        exceptions:
> >            text: "Software implementation requires you to have purchased either Final Fantasy XI or Final Fantasy XIV and registered the game."
> >
> > +    - name: "Star Wars: The Old Republic"		
> > +      url: http://www.swtor.com/		
> > +      img: swtor.png		
> > +      tfa: Yes		
> > +      software: Yes		
> > +      hardware: Yes		
> > +      doc: http://www.swtor.com/info/security-key		
> > +
>
> You have tabs after every line here..
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/2factorauth/twofactorauth/pull/1925/files/6ce8af68888d79bd2b49278d2ce2a510daa05460#r76153542>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABGv4mOrI_WJhD8VdryjHMsbj8Nu_2Qdks5qjMm5gaJpZM4JVpfa>
> .
>